src/jquery.contextMenu.js

```diff
@@ -1089,7 +1089,11 @@
                             $name.append(document.createTextNode(item._afterAccesskey));
                         }
                     } else {
-                        $name.text(item.name);
```

Comment:
I think this might make things weird. Combining the HTML with accessKey will need to make sure the accessKey is not in the HTML parts of the name. Or the decision should be they cant exist together. I'm not entirely sure what is the best approach right now.